install/create_cabinets_yaml.md

```diff
@@ -2,11 +2,32 @@
 
 This page provides directions on constructing the optional `cabinets.yaml` file. This file lists cabinet IDs for any systems with non-contiguous cabinet ID numbers and controls how the `csi config init` command treats cabinet IDs.
 
-The following example file is manually created and follows this format. Each "type" of cabinet can have several fields: `total_number` of cabinets of this type, `starting_id` for this cabinet type, and a list of the IDs.
+The following example file is manually created and follows this format. Each `type` of cabinet can have several fields:
+
+- `total_number` of cabinets of this type.
+- `starting_id` for this cabinet type, and a list of the IDs.
+- `cabinets` to specify individual cabinet overrides, or specify non-contiguous cabinet ranges. These values are optional.
+  - `id` is the cabinet number.
+  - `nmn-vlan` and `hmn-vlan` specify the desired VLAN configuration of the cabinet, instead of allowing Cray Site Init (CSI) to prescribe it.
```

Comment:
@mharding-hpe what is the preferred way to reference CSI? The spell checker does not like `Init`: https://github.com/Cray-HPE/docs-csm/runs/7347744586?check_suite_focus=true

I see other places in docs CSM that reference it the same way I have in this PR:
```
$ grep -R "Cray Site Init (CSI)" .                                                                                                                                                                                                                                              *[CASMHMS-5568]
./install/create_cabinets_yaml.md:  - `nmn-vlan` and `hmn-vlan` specify the desired VLAN configuration of the cabinet, instead of allowing Cray Site Init (CSI) to prescribe it.
./install/create_application_node_config_yaml.md:The `hmn_connections.json` file is derived from the HMN tab of a system SHCD, and is one of the seed files required by Cray Site Init (CSI) command to generate configuration files required to install CSM. The `hmn_connections.json` file is almost a one-to-one copy of the right-hand table in the HMN tab of the SHCD. It is an array of JSON objects, and each object represents a row from the HMN tab. Any row that is not understood by CSI will be ignored, this includes any additional devices connected to the HMN that are not managed by CSM.
./install/configure_management_network.md:Newer systems have HPE Aruba switches, while older systems have Dell and Mellanox switches. Switch IP addresses are generated by Cray Site Init (CSI).
./glossary.md:* [Cray Site Init (CSI)](#cray-site-init)
./glossary.md:## Cray Site Init (CSI)
./glossary.md:The Cray Site Init (CSI) program creates, validates, installs, and upgrades an HPE Cray EX system.
./operations/node_management/Updating_Cabinet_Routes_on_Management_NCNs.md:-   Ensure Cray Site Init (CSI) is installed and available on ncn-m001.
```